src/components/SideNav/index.js

```diff
@@ -6,14 +6,17 @@ import bravoWhite from '../../assets/bravo-white.svg';
 import { SideNavContainer } from './sidenav.styles';
 import Avatar from '../Avatar';
 import SideNavMenu from '../SideNavMenu';
+import TooltipDark from '../Tooltip/Dark';
 
 const SideNav = props => {
   const { slack } = props;
   return (
     <SideNavContainer>
-      <Link to="/">
-        <img src={bravoWhite} alt="Bravo" />
-      </Link>
+      <TooltipDark text="testing">
+        <Link to="/">
+          <img src={bravoWhite} alt="Bravo" />
+        </Link>
+      </TooltipDark>
       {/* { slack.isLoggedIn && <SideNavMenu /> } */}
```

Comment:
should this remain commented out?

Reply:
This was removed in the latest commit and was only for testing